platform/ios/docs/guides/Adding Points to a Map.md

```diff
@@ -63,7 +62,7 @@ To use annotation views, implement `MGLMapViewDelegate` `-mapView:viewForAnnotat
 
 For absolute full control of how points are displayed on a map, consider [runtime styling](runtime-styling.html).
 
-You can use `MGLPointFeature` or any of the other [style feature subclasses](Style%20Features.html) to add points and shapes to an `MGLShapeSource`.
+You can use `MGLPointFeature` or any other [primitive shapes](Primitive%20Shapes.html) to add points and shapes to an `MGLShapeSource`.
```

Comment:
I think this should be a link to [Style Primitives](https://www.mapbox.com/ios-sdk/api/3.6.4/Style%20Primitives.html), not [Primitive Shapes](https://www.mapbox.com/ios-sdk/api/3.6.4/Primitive%20Shapes.html). The former is a more appropriate entry point to runtime styling, whereas the latter is more of an entry point for annotations.